package.json

```diff
@@ -11,7 +11,7 @@
     "health-check": "ts-node src/repos-health-check.ts && tsc",
     "repo-doctor": "ts-node src/repo-doctor.ts && tsc",
     "lint:check": "prettier --check './**'",
-    "lint:fix": "sort-package-json && prettier --write './**'",
+    "lint:fix": "sort-package-json && prettier --write './**.ts' './**.md'",
```

Comment:
why?

Reply:
I think that happened because of the ./** @0xteddybear 

Can you confirm that: `"lint:fix": "sort-package-json && prettier --write .",` still tries to lint the png?